### PR TITLE
deploy bot

### PR DESCRIPTION
**Pull request by Sladuca**
* split workflow into two jobs that run in parallel, one for the api and one for the bot
* created https://github.com/bafnetwork/baf-wallet-deploy-bot
* see it works: https://github.com/bafnetwork/baf-wallet-v2/actions/runs/810965698

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   # This workflow contains a single job called "build"
```
Comment (Lev-Stambler) [inline .github/workflows/deploy.yml:16]: Nit: Update or remove the comment

**Review by Lev-Stambler — CHANGES_REQUESTED** [review]
Looks great! Approved pending change.

Edit: It does not seem to be working in the discord if I do %help. Just rerequest when it gets up and running!

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (Sladuca) [reply .github/workflows/deploy.yml]: just removed the comment, `%help` actually isn't defined

Comment (Sladuca) [reply .github/workflows/deploy.yml]: See, bot works - I just sent you 10 testnet near with it: https://explorer.testnet.near.org/transactions/A6tG27NhDU2DHQG3CKUj5spprMSZeNVaaHGtkxmUwrum

**Review by Lev-Stambler — APPROVED** [review]
Awesome Sauce! It works!

**Pull request merged**